### PR TITLE
Added modules for getting and setting values

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request the following was done: 

1. A new game id was created: 
2. New game scores were entered an pushed to api. 
3. Old game scores were retrieved.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+// add event listener to submit button
+submitBtn.addEventListener('click', () => {
+  postScore(userInput.value, scoreInput.value);
+});
+
+postScore(userInput.value, scoreInput.value);
+
+displayScores();
+
```
Comment (AbelG101) [inline src/index.js]: - [ ] It seems like there's a bug in this code. I noticed that the `postScore()` function is being called outside of the event handler at line `15`, which is causing the user score to be sent to the API multiple times, even though the "Submit" button is only clicked once. This behavior is triggered when the page is refreshed. To fix this issue, I suggest moving the postScore() function inside the "Submit" button's event handler.
- [ ] I'd also suggest moving the `displayScores()` function call inside the event listener for the refresh button

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @dvcorreia1984 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Proper use of `async` and `await` ✔
- UI is looking great ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (dvcorreia1984) [reply src/index.js]: Thank you Abel. I think I managed to fix this bug!!!

**Review by LionRouge1 — APPROVED** [review]
Hi @dvcorreia1984, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**